### PR TITLE
fix: pass only strings to trimIndent

### PR DESCRIPTION
**Pull request by willwill96**
Fixes: https://github.com/keycloakify/keycloakify/issues/285

### CODE REVIEW

**Comment by willwill96** [comment]
I was going to add some unit tests around the jar functionality, but it would have involved mocking some imports which I think is beyond the capabilities of tsafe. Would you accept a PR that adds the ability to test with jest?

File: src/bin/tools/jar.ts [hunk 1]
```diff
@@ -90,5 +90,5 @@ if (require.main === module) {
             groupId: process.env.GROUP_ID ?? "group",
             version: process.env.VERSION ?? "1.0.0"
         });
-    main().catch(e => console.error(e));
```
Comment (willwill96) [inline src/bin/tools/jar.ts:93]: Catching this error was causing keycloakify to swallow errors & instead report a successful message & error code:
image

Comment (garronej) [reply src/bin/tools/jar.ts:93]: Yes 💯  you would **never** catch me doing this, I am a vocal advocate for fail early strategy, I just didn't review.

**Comment by garronej** [comment]
Thank you!! 
You guys rocks. I don't have time to handle it myself but you saved me.

**Comment by garronej** [comment]
> I was going to add some unit tests around the jar functionality, but it would have involved mocking some imports which I think is beyond the capabilities of tsafe. Would you accept a PR that adds the ability to test with jest?   

First of [`tsafe`](https://github.com/garronej/tsafe) isn't a test runner, nor a testing framwork. It's a general pupose utility library.  
In particular, it enable to write assertion of type (assert two types are the same, assert one type extends another) which is extremely usefull to write type level unit test like [this one](https://github.com/keycloakify/keycloakify/blob/main/test/lib/tools/AndByDiscriminatingKey.type.ts) (Test that don't have to be run, it it compiles it passes).  

Now for the runtime test it's stupid not having set a testing framwork from the get go an of course I would accept a PR that do that.  
It would avoid having a a [convoluted process to compile everything in CSJ for testing](https://github.com/keycloakify/keycloakify/blob/3bcc6bdf9315ab7f0811119fa6a52128c8d467bf/package.json#L15).  
That said I would prefer if it was [Vitest](https://vitest.dev/), as this is what I'm used to on my [other](https://github.com/garronej/denoify) [projects](https://github.com/codegouvfr/react-dsfr) but no problem if you are more familiar with Jest, I won't be picky!

**Comment by lordvlad** [comment]
<3

File: src/bin/tools/jar.ts (hunk 1 above)
Comment (lordvlad) [reply src/bin/tools/jar.ts:93]: oh damn, that will affect other placeses too

**Pull request merged**